lib/linguist/languages.yml

```diff
@@ -7909,6 +7909,18 @@ Twig:
   codemirror_mode: twig
   codemirror_mime_type: text/x-twig
   language_id: 377
+Txtx:
+  type: programming
+  color: "#00D992"
+  aliases:
+  - tx
+  extensions:
+  - ".tx"
+  - ".txtx"
+  - ".txvars"
```

Comment:
We require supporting search results and samples for _all_ extensions being added. If any one doesn't meet our usage requirements, it'll hold back the entire PR.

Reply:
Addressed with latest version.